addons/account/models/account_payment.py

```diff
@@ -980,11 +980,13 @@ def _synchronize_to_moves(self, changed_fields):
             pay.move_id \
                 .with_context(skip_invoice_sync=True) \
                 .write({
+                'name' : '/',  # set the name to '/' to make it allowed to change the name
```

Comment:
Why would it change it if it is set?

Reply:
Ok, but where does it change?

Reply:
![image](https://github.com/user-attachments/assets/d761e0c5-9452-4546-a726-6741c87e667a)

that is where payment get a name 
it is the same name as the  journal entry

And that is how the name in being changed in account move 
![image](https://github.com/user-attachments/assets/6c23a06c-3d24-4b52-a5d9-60e7217b213e)


I will add a test now :)

Reply:
@william-andre 
Test added